packages/core/proto/worker.proto

```
@@ -0,0 +1,254 @@
+syntax = "proto3";
+
+package trigger.worker.v1;
```

Buf (1.59.0)

Files with package "trigger.worker.v1" must be within a directory "trigger/worker/v1" relative to root but were in directory "packages/core/proto".

(PACKAGE_DIRECTORY_MATCH)